packages/eas-cli/src/build/android/prepareJob.ts

```diff
@@ -49,11 +50,15 @@ export async function prepareJobAsync(
       }
     : {};
 
-  let buildType = ctx.buildProfile.buildType;
-  if (!buildType && !buildProfile.gradleCommand && ctx.buildProfile.distribution === 'internal') {
+  let buildType = buildProfile.buildType;
+  if (!buildType && !buildProfile.gradleCommand && buildProfile.distribution === 'internal') {
     buildType = Android.BuildType.APK;
   }
 
+  const maybeCustomBuildConfigPath = ctx.buildProfile.config
```

Comment:
```suggestion
  const maybeCustomBuildConfigPath = buildProfile.config
```

Reply:
In some places, you use `buildProfile` and in some `ctx.buildProfile`, which is the same, but it would be good to keep it consistent.